exosphere-shared/example-apps/crashing-service/application.yml

```diff
@@ -6,5 +6,7 @@ services:
   public:
     crasher:
       location: ./crasher
+      docker_image: exospheredev/test-service-that-crashes-on-startup
```

Comment:
That was my initial thought when we were having the discussion on this. The author and title are listed in `service.yml`, because if we are developing a service, we need to identify it. The `docker_image` field of `application.yml` is to tell exosphere what services are being used, whether you have them locally or not, they should be available as docker_images.

Location is needed when using services that are in development, for use in commands like `exo-setup`, `exo-lint`, etc. But the `docker_image` field is needed for actually running. I think this actually exposes an issue for `exo-setup`. I'll put in the ticket now.